wallet/src/de/schildbach/wallet/ExploreSyncWorker.kt

```diff
@@ -117,14 +127,21 @@ class ExploreSyncWorker constructor(val appContext: Context, workerParams: Worke
             }
             val merchantCache = mutableListOf<Merchant>()
             var counter = 0
+            var merchantsLoaded = 0
             exploreRepository.getMerchantData(merchantDataSizeDB).collect {
                 counter++
                 merchantCache.add(it)
                 if (merchantCache.size == PAGE_SIZE) {
                     merchantDao.save(merchantCache)
                     merchantCache.clear()
                     log.info("MERCHANT $counter of ${merchantDataSize - merchantDataSizeDB} sync took $tableSyncWatch")
+
+                    delay(10000)
```

Comment:
This should probably be removed if it was used for testing.

Reply:
Correct.  this is for testing only